src/components/shared/Modal.js

```diff
@@ -19,13 +19,11 @@ export class Modal extends Component {
 
   constructor(props: ModalProps) {
```

Comment:
we won't need the constructor here anymore .. right?

Reply:
Whoops, was on autopilot towards the end and missed it. Fixing now.